### PR TITLE
Istio intro adoc

### PR DESCRIPTION
**Pull request by tt-le**
New common adoc for deployment of istio using `istioctl`

### CODE REVIEW

File: istio-start.adoc [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+```
+
+The output will be similar to the following example.
+[source, role="no_copy"]
+----
+no running Istio pods in "istio-system"
+1.7.3
+----
+
+We will use the following command to configure the {istio} profile on {kube}
```
Comment (gkwan-ibm) [inline istio-start.adoc:23]: @tt-le Should not use the first person words `We`, `I`. `our`, etc. Suggest "We will use..." -> "Use ..."

**Review by gkwan-ibm — CHANGES_REQUESTED** [review]

**Comment by gkwan-ibm** [comment]
@tt-le is `deploy-istio.adoc` better than `istio-start.adoc` ?

File: istio-start.adoc [hunk 2]
```diff
@@ -0,0 +1,67 @@
+// =================================================================================================
+// Deploying Istio
+// =================================================================================================
+
+== Deploying Istio
+
+Install {istio} by following the instructions in the official https://istio.io/latest/docs/setup/getting-started[{istio} Getting started^] documentation.
+
+Run the following command to verify that `istioctl` path has been set successfully:
```
Comment (Charlotte-Holt) [inline istio-start.adoc:9]: Suggested update:

>Run the following command to verify that the `istioctl` path was set successfully:

File: istio-start.adoc [hunk 3]
```diff
@@ -0,0 +1,67 @@
+// =================================================================================================
+// Deploying Istio
+// =================================================================================================
+
+== Deploying Istio
+
+Install {istio} by following the instructions in the official https://istio.io/latest/docs/setup/getting-started[{istio} Getting started^] documentation.
```
Comment (Charlotte-Holt) [inline istio-start.adoc:7]: Include "documentation" in the link text.

File: istio-start.adoc [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+
+Run the following command to verify that `istioctl` path has been set successfully:
+
+[role=command]
+```
+istioctl version
+```
+
+The output will be similar to the following example.
```
Comment (Charlotte-Holt) [inline istio-start.adoc:16]: Add a colon after "The output will be similar to the following example"

File: istio-start.adoc [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+[source, role="no_copy"]
+----
+no running Istio pods in "istio-system"
+1.7.3
+----
+
+Use the following command to configure the {istio} profile on {kube}
```
Comment (Charlotte-Holt) [inline istio-start.adoc:23]: Suggested update

>Run the following command to configure the {istio} profile on {kube}:

File: istio-start.adoc [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+[role=command]
+```
+istioctl install --set profile=demo
+```
+
+The following output should appear when the installation is complete:
```
Comment (Charlotte-Holt) [inline istio-start.adoc:29]: Suggested update

>The following output appears when the installation is complete:

File: istio-start.adoc [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+[source, role="no_copy"]
+----
+NAME                     READY   UP-TO-DATE   AVAILABLE   AGE
+istio-egressgateway      1/1     1            1           2m48s
+istio-ingressgateway     1/1     1            1           2m48s
+istiod                   1/1     1            1           2m48s
+----
+ 
+Ensure that the {istio} deployments are all available before you continue. The deployments might take a few minutes to become available. If the deployments aren't available after a few minutes, then increase the amount of memory available to your {kube} cluster. On Docker Desktop, you can increase the memory from your {docker} preferences. On {minikube}, you can increase the memory using the `--memory` flag.
```
Comment (Charlotte-Holt) [inline istio-start.adoc:57]: Suggested update

>On {minikube}, you can increase the memory using the `--memory` flag.

To

>On {minikube}, you can increase the memory by using the `--memory` flag.

File: istio-start.adoc [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+
+Finally, create the `istio-injection` label and set its value to `enabled`.
```
Comment (Charlotte-Holt) [inline istio-start.adoc:59]: Add a colon after "Finally, create the `istio-injection` label and set its value to `enabled`"

File: istio-start.adoc [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+
+[role=command]
+```
+kubectl label namespace default istio-injection=enabled
+```
+
+Adding this label enables automatic {istio} sidecar injection. Automatic injection means that sidecars will
+automatically be injected into your pods when you deploy your application.
```
Comment (Charlotte-Holt) [inline istio-start.adoc:67]: Suggested update

>Automatic injection means that sidecars will automatically be injected into your pods when you deploy your application.

To

>Automatic injection means that sidecars are automatically injected into your pods when you deploy your application.

**Review by Charlotte-Holt — COMMENTED** [review]
@tt-le I added some minor ID review comments. Let me know if you have any questions!

**Comment by tt-le** [comment]
@Charlotte-Holt Thank you for your feedback, I just addressed it. What do you think now?

**Review by Charlotte-Holt — APPROVED** [review]
@tt-le LGTM, thx!

**Review by gkwan-ibm — APPROVED** [review]

**Pull request merged**